### ISSUE TITLE
[DeepLearning] Homework 4

### ISSUE THREAD

**Issue by eunbin777** [eunbin777/MachineLearning#11]
4.1 Basic AE model
•Generate two autoencoder models for any dataset (not MNIST) with keras API
•with your own AE model (type of autoencoders, model structures, …)
•Print train/validation loss of your own model
•Print original/generated data of your own model
